fc-java-events/src/main/java/com/aliyun/fc/runtime/event/OSSEvent.java

```diff
@@ -1,133 +1,250 @@
-        public final UserIdentity userIdentity;
-
-        @JsonCreator
-        public Event(@JsonProperty("eventName") String eventName,
-            @JsonProperty("eventSource") String eventSource,
-            @JsonProperty("eventTime") String eventTime,
-            @JsonProperty("eventVersion") String eventVersion,
-            @JsonProperty("oss") Oss oss,
-            @JsonProperty("region") String region,
-            @JsonProperty("requestParameters") RequestParameters requestParameters,
-            @JsonProperty("responseElements") ResponseElements responseElements,
-            @JsonProperty("userIdentity") UserIdentity userIdentity) {
+        private String eventName;
+        private String eventSource;
+        private String eventTime;
+        private String eventVersion;
+        private OSSEvent.Event.Oss oss;
+        private String region;
+        private OSSEvent.Event.RequestParameters requestParameters;
+        private OSSEvent.Event.ResponseElements responseElements;
+        private OSSEvent.Event.UserIdentity userIdentity;
+
+        public String getEventName() {
+            return eventName;
+        }
+
+        public void setEventName(String eventName) {
             this.eventName = eventName;
+        }
+
+        public String getEventSource() {
+            return eventSource;
+        }
+
+        public void setEventSource(String eventSource) {
             this.eventSource = eventSource;
+        }
+
+        public String getEventTime() {
+            return eventTime;
+        }
+
+        public void setEventTime(String eventTime) {
             this.eventTime = eventTime;
+        }
+
+        public String getEventVersion() {
+            return eventVersion;
+        }
+
+        public void setEventVersion(String eventVersion) {
             this.eventVersion = eventVersion;
+        }
+
+        public Oss getOss() {
+            return oss;
+        }
+
+        public void setOss(Oss oss) {
             this.oss = oss;
+        }
+
+        public String getRegion() {
+            return region;
+        }
+
+        public void setRegion(String region) {
             this.region = region;
+        }
+
+        public RequestParameters getRequestParameters() {
+            return requestParameters;
+        }
+
+        public void setRequestParameters(
+            RequestParameters requestParameters) {
             this.requestParameters = requestParameters;
+        }
+
+        public ResponseElements getResponseElements() {
+            return responseElements;
+        }
+
+        public void setResponseElements(
+            ResponseElements responseElements) {
             this.responseElements = responseElements;
+        }
+
+        public UserIdentity getUserIdentity() {
+            return userIdentity;
+        }
+
+        public void setUserIdentity(UserIdentity userIdentity) {
             this.userIdentity = userIdentity;
         }
 
```

Comment:
Can you implement `equals`, `hashCode` and `toString` methods? Those would be useful to write test and print.